api/v1/server/handlers/v1/webhooks/receive.go

```diff
@@ -131,10 +214,17 @@ func (w *V1WebhooksService) V1WebhookReceive(ctx echo.Context, request gen.V1Web
 	)
 
 	if err != nil {
-		if eventKey == "" {
-			err = fmt.Errorf("event key evaluted to an empty string")
+		var errorMsg string
+		if strings.Contains(err.Error(), "did not evaluate to a string") {
+			errorMsg = fmt.Sprintf("event key expression must evaluate to a string, but got a different type. Expression: %s. Error: %v. For Slack webhooks, try: 'slack:' + input.type or input.type", webhook.EventKeyExpression, err)
```

Comment:
do we want slack specifics in these error messages?

Reply:
Nope good call removed! https://github.com/hatchet-dev/hatchet/pull/2612/commits/8ab18e6ffcda578bfa9e155dbcdec72c4102e71c